tests/static/js/jqunit/jqunit-binder-array.js

```diff
@@ -0,0 +1,45 @@
+/* globals fluid */
```

Comment:
Why are all these tests in a directory named "static"? Is there an alternative to static tests?